packages/arb-util/ethutils/client.go

```diff
@@ -78,12 +78,17 @@ func NewRPCEthClient(url string) (*RPCEthClient, error) {
 func (r *RPCEthClient) reconnect() error {
 	r.Lock()
 	defer r.Unlock()
+	if r.errCount < maxErrCount {
```

Comment:
Hmm.. technically, this (and the set below) need to also be atomic as the error count is atomically written to outside the mutex.

Reply:
Ah whoops. Good catch, I hadn't been thinking. Fix pushed